kong/custom_entity_service.go

```diff
@@ -35,7 +35,7 @@ func (s *CustomEntityService) Get(ctx context.Context,
 	entity custom.Entity) (custom.Entity, error) {
 	def := s.client.Lookup(entity.Type())
 	if def == nil {
-		return nil, errors.New("entity '" + string(entity.Type()) +
+		return nil, fmt.Errorf("entity '" + string(entity.Type()) +
```

Comment:
Would it be an opportunity to use verbs here and in the future rest of the file ?